spec/passaporte_web/service_account_member_spec.rb

```diff
@@ -38,7 +38,7 @@
     end
     it "should raise an 404 error if the membership does not exist" do
       expect {
-        described_class.find(service_account, mock('Identity', uuid: 'identity-uuid'))
+        described_class.find(service_account, double('Identity', uuid: 'identity-uuid'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.